Bottom: 2a540220b1caae6d593fb8381d7c99b70d20da7d
Top:    4f5b25f0a23297c448d6a14e6361a94bd795288d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 01:18:39 +0100

Refresh of hid-gtk-remove-auto-pan-featur

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index b213d54..20e8e03 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -148,45 +148,6 @@ ghid_note_event_location (GdkEventButton * ev)
   return moved;
 }
 
-gboolean
-have_crosshair_attachments (void)
-{
-  gboolean result = FALSE;
-
-  switch (Settings.Mode)
-    {
-    case COPY_MODE:
-    case MOVE_MODE:
-    case INSERTPOINT_MODE:
-      if (Crosshair.AttachedObject.Type != NO_TYPE)
-	result = TRUE;
-      break;
-    case PASTEBUFFER_MODE:
-    case VIA_MODE:
-      result = TRUE;
-      break;
-    case POLYGON_MODE:
-    case POLYGONHOLE_MODE:
-      if (Crosshair.AttachedLine.State != STATE_FIRST)
-	result = TRUE;
-      break;
-    case ARC_MODE:
-      if (Crosshair.AttachedBox.State != STATE_FIRST)
-	result = TRUE;
-      break;
-    case LINE_MODE:
-      if (Crosshair.AttachedLine.State != STATE_FIRST)
-	result = TRUE;
-      break;
-    default:
-      if (Crosshair.AttachedBox.State == STATE_SECOND
-	  || Crosshair.AttachedBox.State == STATE_THIRD)
-	result = TRUE;
-      break;
-    }
-  return result;
-}
-
 static gboolean
 ghid_idle_cb (gpointer data)
 {
diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index e3fe8b0..5ba5aeb 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -277,7 +277,6 @@ void ghid_port_ranges_changed (void);
 void ghid_port_ranges_scale (void);
 
 gboolean ghid_note_event_location (GdkEventButton * ev);
-gboolean have_crosshair_attachments (void);
 gboolean ghid_port_key_press_cb (GtkWidget * drawing_area,
 				 GdkEventKey * kev, gpointer data);
 gboolean ghid_port_key_release_cb (GtkWidget * drawing_area,
